requirements.sweeps.txt

```diff
@@ -0,0 +1,7 @@
+numpy>=1.15,<1.21
+scipy>=1.5.4
+PyYAML
+scikit_learn==0.24.1
+jsonschema>=3.2.0
+jsonref>=0.2
+pydantic>=1.8.2
```

Comment:
the contents of this file is vendored using `bash tools/vendor_sweeps.sh`. It is automatically verified in circle using `tox -e verify-sweeps`.